### DISCUSSION TITLE
MPS (Mathematical Programming System) misclassified as Jet Brains MPS

### DISCUSSION THREAD

**Discussion by JasonMarechal25** [github-linguist/linguist#6129] [Classification] [answered]
In our project our data are defined with Mathematical Programming System (MPS) format: https://en.wikipedia.org/wiki/MPS_(format)
This format is misclassified as JetBrains MPS format

Link to repository https://github.com/AntaresSimulatorTeam/antares-xpansion
Link to file example https://github.com/AntaresSimulatorTeam/antares-xpansion/blob/develop/data_test/mini_instance_LP/master.mps

**Comment by lildude** [comment, marked as answer]
This will be because Linguist doesn't know anything about Mathematical Programming System and as such all users of the `.mps` extension are associated with JetBrains MPS.

If you would like to add support, please submit a PR adding support, or please open a new issue using the correct template.
